net-vpn/i2pd/i2pd-2.49.0.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="I2P: End-to-End encrypted and anonymous Internet"
+DESCRIPTION="i2pd (I2P Daemon) is a full-featured C++ implementation of I2P client.\
+I2P (Invisible Internet Protocol) is a universal anonymous network layer.\
+All communications over I2P are anonymous and end-to-end encrypted, participants \
+don't reveal their real IP addresses.\
+I2P client is a software used for building and using anonymous I2P networks. \ 
+Such networks are commonly used for anonymous peer-to-peer applications \
+(filesharing, cryptocurrencies) and anonymous client-server applications \ 
+(websites, instant messengers, chat-servers).\
+I2P allows people from all around the world to communicate and share information without restrictions.
+"
+HOMEPAGE="https://i2pd.website/"
+COPYRIGHT="i2pd developers"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/PurpleI2P/i2pd/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="d8e07f78cc9f1ba65e8460db27c649dd0cfdd3ba334725f8d6f9ee815cb40e68"
+PATCHES="$portName-$portVersion.patchset"
```

Comment:
As mentioned in the previous PR, don't use `$portName` here. This failed last time on buildmaster 32bit also.